packages/contracts-bedrock/scripts/checks/common/util.go

```diff
@@ -48,13 +48,13 @@ func ProcessFiles[T any](files map[string]string, processor FileProcessor[T]) (m
 	reporter := NewErrorReporter()
 	results := sync.Map{}
 
-	for name, path := range files {
-		name, path := name, path // Capture loop variables
+	for _, path := range files {
```

Comment:
Can we undo this diff?

Reply:
Added it because it makes the path of the file in question clear, some files have the same name but in different folders and can be confusing to know which is being referenced